### ISSUE TITLE
Client group checklist fix

### ISSUE THREAD

**Issue by markstreaka** [streaka/plantguard-issues#299]
Please see this video: http://streaka.testing.videos.s3.amazonaws.com/screencast_2017-06-21_22-11-33.mp4
And see this image: http://streaka.testing.videos.s3.amazonaws.com/PlantGUARD_-_Checklists_2017-06-21_22-22-18.png

**Comment by alexkingma1** [comment]
I've had a look just now, and the main issue seems to be to do with the ability to assign checklists, which I removed (most of) today, but which was still blocking a bit of functionality. This is why the checklists aren't showing up to that first schwarz plant in the Begin Checklist dropdown, even when made public.

As for what the public/private designations mean, I intend to give each field a tooltip once I've rolled out the new table changes to the checklist tables. As I see it, private means only the business for which it was created can use/see it; public means anyone in the clientGroup can use/see it. So in this case with the 3 aurizon checklists, the first two would be public (usable by all users/subusers within Aurizon's account AND all suppliers in the Aurizon client group), while the third would be private (only available to Aurizon users/subusers, and NOT their suppliers or in fact anyone else on the PlantGUARD system).

If this isn't how you see it, we can chat about it tomorrow.
